types/dummy/index.d.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
Do we need this empty file? What does it do for us?

Reply:
We don't! I'll remove it!